### ISSUE TITLE
I modified the file about gradle and successfully built it

### ISSUE THREAD

**Issue by Origin-GreenCat** [kadir1243/mcreator-generator-1.7.10#1]
I modified the file about gradle to build it successfully, but I can't write mod of 1.7.10, so I can't modify the ftl file, so I can't successfully execute the gradle task: runClient

**Comment by kadir1243** [comment]
hello GreenCat
this generator is still have a lot of error
if you know java you can create pull request for fix errors

**Comment by Origin-GreenCat** [comment]
Yes,I do know Java, and I have written Java programs and a new version of Minecraft mod, but I don't know how to make the 1.7.10 version mod ,I don't know how to write Mcreator plug-in, so I can't make this plug-in become，Become available。And I'm still in China, so it's very difficult to build gradle or rely on downloading, so I can't do plug-in test conveniently.------------------&nbsp;原始邮件&nbsp;------------------
***@***.***&gt;
发送时间:&nbsp;2021年6月3日(星期四) 下午5:33
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [kadir1243/mcreator-generator-1.7.10] I modified the file about gradle and successfully built it (#1)

**Comment by kadir1243** [comment]
it's using freemarker like other generators
freemarker(mcreator generators using freemarker) is just java and html languages and not difficult
and if you want to write 1.7.10 mod you can use this template https://github.com/anatawa12/ForgeGradle-example

**Comment by Origin-GreenCat** [comment]
Ok，I try it.

**Comment by WyattTheSkid** [comment]
is this dead? I will pay you to finish this if you have abandoned it.

**Comment by WyattTheSkid** [comment]
> I modified the file about gradle to build it successfully, but I can't write mod of 1.7.10, so I can't modify the ftl file, so I can't successfully execute the gradle task: runClient

how did you built it?
